sal-service/src/main/java/org/ow2/proactive/sal/service/service/RepositoryService.java

```diff
@@ -287,6 +287,7 @@ public Deployment deleteDeployment(String deploymentId) {
     @Modifying(clearAutomatically = true)
     public Deployment deleteDeployment(Deployment deployment) {
         // remove deployments from paCloud
+        LOGGER.info("Cleaning deployment \"{}\" from the cloud entry and related tasks", deployment.getNodeName());
```

Comment:
Maybe split this log in two if you think it is innteresting:
`Cleaning deployment \"{}\" from the related cloud`
`Cleaning deployment \"{}\" from the related task`

Reply:
done